From: Tias Guns <tias@ulyssis.org>
Date: Mon, 19 Mar 2012 18:14:44 +0000
Subject: [PATCH] fix inetd: sys/un.h needs sys/socket.h included

patch from 'sys-un.h-needs-sys-socket.h' by Dan Drown
http://dan.drown.org/android/src/busybox/
---
 networking/inetd.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/networking/inetd.c b/networking/inetd.c
index 26b6699..589c267 100644
--- a/networking/inetd.c
+++ b/networking/inetd.c
@@ -165,6 +165,7 @@
 //usage:     "\n		(default: 0 - disabled)"
 
 #include <syslog.h>
+#include <sys/socket.h>
 #include <sys/un.h>
 
 #include "libbb.h"
-- 
1.7.0.4

